schema/in_memory_entity/has_consistency_checks.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$id": "in-memory-entity/has-consistency-checks",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Has consistency checks",
+    "allOf": [
+        {
+            "$ref": "base.json"
+        },
+        {
+            "$ref": "../system/consistency_checks.json"
```

Comment:
Let's just add into the in-memory entity iself

Reply:
That means to the `in_memory_entity/base.json`